dubbo-filter/dubbo-filter-cache/src/main/java/org/apache/dubbo/cache/support/expiring/ExpiringMap.java

```diff
@@ -263,7 +264,7 @@ public String toString() {
         }
 
         public ExpireThread() {
-            expirerThread = new Thread(this, "ExpiryMapExpire-" + expireCount++);
+            expirerThread = new Thread(this, "ExpiryMapExpire-" + expireCount.getAndIncrement());
```

Comment:
expireCount.incrementAndGet()

Reply:
@lixiaojiee Please test, i++ means to use first, then +1;

Reply:
My fault